### DISCUSSION TITLE
Autofill login form Keycloak

### DISCUSSION THREAD

**Discussion by MaxiRage** [keycloak/keycloak#23617] [Q&A] [answered]
I didn't find a way for the browser to autofill the login and password in the Keycloak login form
Has anyone been interested and solved this issue?
By replacing "autofill =off" with "=on" in the HTML file, the browser will display a list of selecting only the user name

**Comment by rene6502** [comment]
For me this works out of the box (tested with Chrome/Chromium/Firefox/Edge).
I have stored the username and password in the browser’s password manager.
I guess https is also mandatory for this to work.

**Reply by MaxiRage** [reply]
I also assuming it would work out of the box.
It does work on http, but not on https.
When I try to manually add to the password manager, I get an error in the browser console:

![photo_2023-10-02_11-28-17](https://github.com/keycloak/keycloak/assets/87515511/ac57b45a-11d1-4fee-92d1-cdaa6216ad5f)

**Comment by rene6502** [comment]
Isn’t that more a problem on the browser side than on the Keycloak side?
I assume you are using Chrome. [Here](https://source.chromium.org/) you can look into the source code.
I would try to update to the latest version and try to reset the profile (backup before) or add a new profile.

**Reply by MaxiRage** [reply]
This behavior is observed in different users

**Comment by MaxiRage** [comment, marked as answer]
After assigning dns and issuing the certificate, the problem went away.
